coalib/bearlib/aspects/base.py

```diff
@@ -1,8 +1,94 @@
+import re
+
 from coalib.bearlib.languages import Language
 
+import coalib.bearlib.aspects
 from .taste import TasteError
 
 
+class SubaspectGetter:
+    """
+    Special "getter" class to implement get() function in aspectbase that
+    could be accessed from the aspectclass or aspectclass instance.
+    """
+
+    def __get__(self, obj, owner):
+        if obj is not None:
+            parent = obj
+            parent_qualname = type(obj).__qualname__
+        else:
+            parent = owner
+            parent_qualname = owner.__qualname__
+
+        def get_subaspect(subaspect):
```

Comment:
This inner method has become pretty huge, so better move it to global module level, giving it an additional parent aspect attribute